File: easy/removeelement.cpp
#include <iostream>

class Solution {
public:
    int removeElement(vector<int>& nums, int val) {
        int count = 0, temp;
        for(int i = 0; i < nums.size(); i++) {
            if(nums[i] != val) {
                temp = nums[i];
                nums[i] = nums[count];
                nums[count] = temp;
                count += 1;
            }
        }

        return count;
    }
};

File: python/easy/removeelement.py
class Solution:
    def removeElement(self, nums: List[int], val: int) -> int:
        count = 0
        n = len(nums)
        for i in range(n):
            if(nums[i] != val):
                temp = nums[i]
                nums[i] = nums[count]
                nums[count] = temp
                count += 1

        return count